src/main/java/com/ays/admin_user/repository/AdminUserRepository.java

```diff
@@ -43,6 +43,6 @@ public interface AdminUserRepository extends JpaRepository<AdminUserEntity, Stri
      * @param lineNumber  the phone number of the user to check
      * @return true if an {@link AdminUserEntity} exists with the given country code and phone number, false otherwise
      */
-    boolean existsByCountryCodeAndLineNumber(Long countryCode, Long lineNumber);
+    boolean existsByCountryCodeAndLineNumber(String countryCode, String lineNumber);
 
 }
```

Comment:
Bu sınıf içerisindeki geliştirmelere ihtiyaç kalmadığı için geri alabilir miyiz?